### ISSUE TITLE
Support for UNLINK

### ISSUE THREAD

**Issue by govindbalaji-s** [fppt/jedis-mock#292]
https://redis.io/commands/unlink/
I can't see how this can be made to behave any differently than DEL in java. But I feel even an implementation that just calls DEL instead will also be helpful. I can go ahead and implement this, if this is fine.

**Comment by inponomarev** [comment]
Hi, thanks for the suggestion! AFAICS `UNLINK` is there just for better performance, so for testing purposes an alias for `DEL` will do. It is trivial, I implemented it in #293 

Expect the release soon.

As a temporary workaround, I suggest you to use a `RedisCommandInterceptor` (see README) with the following:

```java
(state, roName, params) -> {
        if ("unlink".equalsIgnoreCase(roName)) {
            return MockExecutor.proceed(state, "del", params);
        } else {
            return MockExecutor.proceed(state, roName, params);
        }
```

**Comment by govindbalaji-s** [comment]
Thanks Ivan. I have used the same interceptor for now.
